packages/@aws-cdk/aws-s3/package.json

```diff
@@ -90,6 +90,7 @@
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
   "awslint": {
+    "flag": [ "use-constructs-module" ],
```

Comment:
I'd simply call this `include` so it's canonic with `exclude`

Reply:
`include` already exists and indicates the rules to opt into.

Reply:
Yeah forget about it